website/cue/reference/releases/0.33.1.cue

```diff
@@ -0,0 +1,70 @@
+package metadata
+
+releases: "0.33.1": {
+	date:     "2023-10-30"
+	codename: ""
+
+	whats_next: []
+
+	description: """
+		This patch release contains fixes for regressions in 0.33.0 and fixes an issues with the Debian release artifacts.
+
+		**Note:** Please see the release notes for [`v0.33.0`](/releases/0.33.0/) for additional changes if upgrading from
+		`v0.32.X`. In particular, the upgrade guide for breaking changes.
+		"""
+
+	changelog: [
+		{
+			type: "fix"
+			scopes: ["releasing", "debian"]
+			description: """
+				Debian packages again avoid overwriting existing configuration files when upgrading.
+				"""
+			pr_numbers: [18718]
+		},
+		{
+			type: "fix"
+			scopes: ["datadog_metrics sink"]
+			description: """
+				The performance of the Datadog Metrics sink was greatly improved when the incoming
+				metric stream contains mostly counters.
+				"""
+			pr_numbers: [18759]
+		},
+		{
+			type: "fix"
+			scopes: ["dnstap source"]
+			description: """
+				The `dnstap` source can again parse DNSSEC/RRSIG RRs records.
+				"""
+			pr_numbers: [18878]
+		},
+		{
+			type: "fix"
+			scopes: ["kafka source"]
+			description: """
+				A performance regression in the `kafka` source was corrected.
```

Comment:
fixed component is kafka sink, not source

Reply:
Thanks for catching that! I opened https://github.com/vectordotdev/vector/pull/19032 to correct it.